cases/models.py

```diff
@@ -509,6 +521,21 @@ def number_all_complaints(self):
     def number_all_complainants(self):
         return len({c.complainant_id for c in self.all_complaints_reversed})
 
+    @property
+    def file_storage_used_bytes(self):
+        bytes_used = 0
+        for action in self.actions.all():
```

Comment:
This double loop will also involve a database query for each action on the case. I think if the first line became `self.actions.prefetch_related('files')`, then that should have prefetched the files for each action.